gemfiles/rails4.2.gemfile

```diff
@@ -7,15 +7,17 @@ gem "aruba"
 gem "coffee-rails"
```

Comment:
The Gemfiles in the `gemfiles` directory are generated by Appraisal, so we should just have RuboCop ignore them like we do here: https://github.com/thoughtbot/factory_bot/commit/a56b68bc9c552064c8551b82945d92b647b25932#diff-11a0d7906801d9dea0eccb85667ad811R3

Reply:
makes sense, I reset my changes and excluded `gemfiles/*` here: https://github.com/thoughtbot/factory_bot_rails/pull/295/commits/1047509c61efd5e3c87f34c4d4ecf24cadb0eaba